bool limit_switch_up
bool limit_switch_down
